lib/rules/no-v-html.js

```diff
@@ -14,16 +15,32 @@ module.exports = {
       url: 'https://eslint.vuejs.org/rules/no-v-html.html'
     },
     fixable: null,
-    schema: [],
+    schema: [{
+      type: 'object',
+      properties: {
+        ignorePattern: {
+          type: 'string'
+        }
+      },
+      additionalProperties: false
+    }],
     messages: {
       unexpected: "'v-html' directive can lead to XSS attack."
```

Comment:
I wonder if a separate message would make sense if `ignorePattern` is defined… what do you think?

Reply:
I don't think that is necessary.